### ISSUE TITLE
Ansible Silo - probably is in the wrong section

### ISSUE THREAD

**Issue by it-praktyk** [jdauphant/awesome-ansible#26]
I'm not 100% sure but IMHO Ansible Sillo should be moved to the Tool section.

@udondan, what do you think?

**Comment by udondan** [comment]
I agree. When there have been other links in the Docker section it made sense to list it there. But now that it is the only remaining link... I think we can remove the Docker section and move it to the tools. It certainly is a tool, that just happens to build on Docker.
